src/java.base/share/native/libsyslookup/syslookup.c

```diff
@@ -27,6 +27,11 @@
 // Adding at least one #include removes unwanted warnings on some platforms.
 #include <stdlib.h>
 
+#include <jni.h>
```

Comment:
Do we need this include? DEF_STATIC_JNI_OnLoad is defined by jni_util.h, which should include jni.h?

Reply:
> Do we need this include? DEF_STATIC_JNI_OnLoad is defined by jni_util.h, which should include jni.h?

Removed, thanks!